src/client/signatures.rs

```diff
@@ -159,13 +153,13 @@ pub enum SignatureError {
     #[error("certificate could not be downloaded from {}: HTTP {}", response.url, response.status)]
     CertificateDownloadError { response: Response },
     #[error("certificate content could not be parsed: {0}")]
-    CertificateContentError(#[from] x509::X509Error),
+    CertificateContentError(String),
     #[error("root certificate fingerprint has bad format: {0}")]
-    RootFormatError(String),
+    RootHashFormatError(String),
```

Comment:
For what it's worth, I've had a lot of success combining *both* `thiserror` and `anyhow`. The strings here aren't intended to be machine-readable. Using `anyhow` lets you preserver more context while keeping the free form nature that strings provide.

Probably not worth changing in this PR, but maybe something to consider for 1.0?

Reply:
Thanks! I created https://github.com/mozilla-services/remote-settings-client/issues/143